tmd/Topology/vectorizations.py

```diff
@@ -0,0 +1,112 @@
+"""Topology vectorization algorithms."""
+
+# Copyright (C) 2022  Blue Brain Project, EPFL
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# pylint: disable=invalid-slice-index
+import copy
+
+import numpy as np
+from scipy import stats
+
+from .statistics import get_lengths
+
+
+def get_limits(phs_list):
```

Comment:
The `collapse` feature is no more needed here?

Reply:
not sure why it was there in the first place :/ I think it was bad implementation.

Reply:
ok, wrong assessment, I rewrote that part because it failed in some cases as it is.

Reply:
ok :)